src/buttons.css

```diff
@@ -174,12 +168,14 @@ active color is visible on all borders */
 /* Extra .btn in selectors below to increase specificity
 against .round class */
 
-/* 1px margins create gutters. -2px margins blend borders */
+/* -3px margins on horizontal pills work around space created
+between inline elements, while maintaining gutters */
```

Comment:
I think this is a real problem ... the space can vary based on browser, font, and maybe some other factors. This was one reason to use a flex parent. Why do you think we should remove the flex parent?